### PR TITLE
fix: upgrade Multi Input to fundamental-styles 0.6.0

### PR DESCRIPTION
**Pull request by mikerodonnell89**
#### Please provide a link to the associated issue.

#2008 

#### Please provide a brief summary of this pull request.

Reopening due to travis issue

### CODE REVIEW

File: libs/core/src/lib/checkbox/checkbox/checkbox.component.ts [hunk 1]
```diff
@@ -86,7 +86,8 @@ export class CheckboxComponent implements ControlValueAccessor {
     public onValueChange = (newValue) => {};
 
     constructor(
```
Comment (InnaAtanasova) [inline libs/core/src/lib/checkbox/checkbox/checkbox.component.ts]: Should have `hidden`

File: libs/core/src/lib/list/list-item.directive.ts [hunk 2]
```diff
@@ -27,4 +28,25 @@ export class ListItemDirective {
     @HostBinding('class.is-selected')
     selected: boolean = false;
 
+    @ContentChild(CheckboxComponent, {static: false})
```
Comment (InnaAtanasova) [inline libs/core/src/lib/list/list-item.directive.ts]: Should have `hidden`

File: libs/core/src/lib/list/list-item.directive.ts [hunk 3]
```diff
@@ -27,4 +28,25 @@ export class ListItemDirective {
...
+    checkboxComponent: CheckboxComponent;
+
+    /** @hidden */
+    constructor (public itemEl: ElementRef) {}
+
+    public focus(): void {
```
Comment (InnaAtanasova) [inline libs/core/src/lib/list/list-item.directive.ts]: Typedocs missing for this method

File: libs/core/src/lib/list/list-item.directive.ts [hunk 4]
```diff
@@ -27,4 +28,25 @@ export class ListItemDirective {
...
+        if (this.checkboxComponent) { // if there is a checkbox in this list item, we want to focus its input label
+            this.checkboxComponent.inputLabel.nativeElement.focus();
+        } else {
+            this.itemEl.nativeElement.focus();
+        }
+    }
+
+    public click(): void {
```
Comment (InnaAtanasova) [inline libs/core/src/lib/list/list-item.directive.ts]: Typedocs missing for this method

File: libs/core/src/lib/multi-input/multi-input.component.spec.ts [hunk 5]
```diff
@@ -9,8 +9,9 @@ import { PopoverModule } from '../popover/popover.module';
 import { PipeModule } from '../utils/pipes/pipe.module';
 import { InputGroupModule } from '../input-group/input-group.module';
 import { CheckboxModule } from '../checkbox/checkbox.module';
+import { ListModule } from '../list/list.module';
 
-describe('MultiInputComponent', () => {
+fdescribe('MultiInputComponent', () => {
```
[the remark this answers is not in the document]
Comment (mikerodonnell89) [reply libs/core/src/lib/multi-input/multi-input.component.spec.ts]: oops good catch

File: libs/core/src/lib/list/list-item.directive.ts (hunk 3 above)
Comment (mikerodonnell89) [reply libs/core/src/lib/list/list-item.directive.ts]: These are just convenience methods, so i'll @hidden them

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-02-28 at 10 04 07 AM

The Input field in the Compact example is not compact

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-02-28 at 10 05 01 AM

Could you add some margin bottom to the input in the docs just to make it more presentable. Thanks

**Comment by mikerodonnell89** [comment]
> Screen Shot 2020-02-28 at 10 04 07 AM
> 
> The Input field in the Compact example is not compact

this must be a netlify AOT thing.  but jedrjez should me how to avoid it on our end, i'll give it a shot

**Comment by mikerodonnell89** [comment]
> Screen Shot 2020-02-28 at 10 02 13 AM
> 
> I just noticed that there's no space between the Show All btn and the last item of the list

Do you mean no border?  That's how it is in the specs

**Pull request merged**